tests/TestCase.php

```diff
@@ -2,9 +2,10 @@
 
 namespace Tests;
 
+use Illuminate\Foundation\Testing\DatabaseTransactions;
 use Illuminate\Foundation\Testing\TestCase as BaseTestCase;
 
 abstract class TestCase extends BaseTestCase
 {
-    use CreatesApplication;
+    use CreatesApplication, DatabaseTransactions;
```

Comment:
No need for DatabaseTransaction trait, instead of this trait you may need use DatabaseRefresh in the CustomerControllerTest